cernopendata/modules/fixtures/data/records/jade-data.json

```diff
@@ -0,0 +1,78 @@
+[
+  {
+    "abstract": {
+      "description": "<p>JADE real data</p>",
+      "links": [
+        {
+          "description": "All JADE runs",
+          "recid": "FIXME"
+        }
+      ]
+    },
+    "accelerator": "DESY-PETRA",
+    "collaboration": {
+      "name": "JADE collaboration",
+      "recid": "26050"
+    },
+    "collections": [
+      "JADE-Primary-Datasets"
+    ],
+    "collision_information": {
+      "energy": "12GeV",
+      "type": "e+e-"
+    },
+    "date_created": [
+      "1986"
+    ],
+    "date_published": "2024",
+    "date_reprocessed": "1986",
+    "distribution": {
+      "formats": [
+        "raw"
+      ],
+      "number_events": 7017835,
+      "number_files": 2861,
+      "size": 2614994979
+    },
+    "doi": "10.7483/OPENDATA.9S5F.BY3B",
+    "experiment": [
+      "JADE"
+    ],
+    "files": [
+      {
+        "checksum": "adler32:44e5147f",
+        "size": 1575978,
+        "type": "index.json",
+        "uri": "root://eospublic.cern.ch//eos/opendata/jade/tapes_index.json"
+      }
+    ],
+    "keywords": [
+      "e+e- physics"
+    ],
+    "license": {
+      "attribution": "CC0"
+    },
+    "methodology": {
+      "description": "<p> During the experiment, the raw data was written to the tapes.</p>"
+    },
+    "publisher": "CERN Open Data Portal",
+    "recid": "FIXME",
+    "title": "JADE raw data",
```

Comment:
Please expand the title, it seems a bit too brief, especially considering that there will be only one record for all tapes. Something like "JADE detector raw data from electron-positron collision events at the PETRA accelerator".  (Quoting some text just for inspiration...)

Reply:
This comment still holds, e.g. currently we have on the search pages:

![jadetitle1](https://github.com/user-attachments/assets/f2d4dc1c-998b-4397-9cae-c74a92b47243)

And on the detailed record page:

![jadetitle2](https://github.com/user-attachments/assets/a2e0c722-4273-450e-abbb-4461bd785282)

It may not be necessary to use both `title` and `title_additional` fields. This was mostly useful for CMS which uses this field to store the exact dataset title in machine-readable format.  For the old JADE data, it would be enough to have only one `title` field, so that the title information in searches and on the detailed record page would be identical.

WDYT?

Reply:
Sounds good, I moved the `title_additional ` to the main `title`.